firefox-ios/Client/Frontend/Browser/Tabs/Middleware/TabManagerMiddleware.swift

```diff
@@ -1237,27 +1261,23 @@ final class TabManagerMiddleware: FeatureFlaggable {
         profile.pinnedSites.removeFromPinnedTopSites(site)
     }
 
+    @MainActor
     private func preserveTabs(uuid: WindowUUID) {
         let tabManager = tabManager(for: uuid)
         tabManager.preserveTabs()
     }
 
     /// Sends out updated recent tabs which is currently used for the homepage jumpBackIn section
+    @MainActor
     private func dispatchRecentlyAccessedTabs(action: Action) {
-        // TODO: FXIOS-10919 - Consider testing better with Tasks here
-        // and modifying how we fetch recentlyAccessedNormalTabs since
-        // it doesn't retrieve the proper tabs without this task block
-        // See more details on issue here [FXIOS-5149] [FXIOS-11644]
-        Task { @MainActor in
-            // [FXIOS-5149] Recent tabs need to be accessed from .main thread
-            let recentTabs = self.tabManager(for: action.windowUUID).recentlyAccessedNormalTabs
-            store.dispatchLegacy(
-                TabManagerAction(
-                    recentTabs: recentTabs,
-                    windowUUID: action.windowUUID,
-                    actionType: TabManagerMiddlewareActionType.fetchedRecentTabs
-                )
+        // [FXIOS-5149] Recent tabs need to be accessed from .main thread
```

Comment:
``` 
// [FXIOS-5149] Recent tabs need to be accessed from .main thread
```
nit; Is that comment still relevant? This was added as part of the jump back in work a long time ago, and the tab manager was in a very different state than it was today. Since everything is basically `@MainActor`, does it add value to have this comment?